docs/ed-fi-badges/2-badging-requirements-for-technology-partners.md

```diff
@@ -0,0 +1,35 @@
+---
+author: Ecleamus Ricks
```

Comment:
In general, we've only been listing authors on blog posts, not on regular content pages.